common/defines.lua

```diff
@@ -198,21 +198,21 @@ military = {
 diplomacy = {
 	PEACE_COST_ADD_TO_SPHERE = 1,
 	PEACE_COST_RELEASE_PUPPET = 1,
-	PEACE_COST_MAKE_PUPPET = 1,
+	PEACE_COST_MAKE_PUPPET = 85,
 	PEACE_COST_DISARMAMENT = 1,
 	PEACE_COST_DESTROY_FORTS = 1,
 	PEACE_COST_DESTROY_NAVAL_BASES = 1,
 	PEACE_COST_REPARATIONS = 1,
 	PEACE_COST_TRANSFER_PROVINCES = 1,
-	PEACE_COST_REMOVE_CORES = 1,
+	PEACE_COST_REMOVE_CORES = 0,
 	PEACE_COST_PRESTIGE = 1,
 	PEACE_COST_CONCEDE = 1,
 	PEACE_COST_STATUS_QUO = 1,
-	PEACE_COST_ANNEX = 1,
-	PEACE_COST_DEMAND_STATE = 1,
-	PEACE_COST_INSTALL_COMMUNIST_GOV_TYPE = 1,
+	PEACE_COST_ANNEX = 85,
+	PEACE_COST_DEMAND_STATE = 10,
+	PEACE_COST_INSTALL_COMMUNIST_GOV_TYPE = 70,
 	PEACE_COST_UNINSTALL_COMMUNIST_GOV_TYPE = 1,
-	PEACE_COST_COLONY = 1,
+	PEACE_COST_COLONY = 10,
```

Comment:
Have you tested all of the cbs that this affects? I specifically made them all 1 to make changing and predicting the peace cost easier.

I'm going to need to see an example for each of the standard cbs that use these POs with the old peace cost in place, compared to the same cb (targetting the same country and region, as applicable) with the new peace cost in place